Dockerfile

```diff
@@ -1,53 +1,63 @@
-FROM debian:jessie
+FROM erlang:20.3 AS build-env
 
-MAINTAINER Octavo Labs AG info@vernemq.com
+WORKDIR /tmp/vernemq-build
```

Comment:
Now that I'm looking at this - any reason why you're using /tmp here? Out of the top of my head, /tmp might be mounted as ramfs, and I'm not sure if that applies to Docker containers as well or if it might cause any issues in some environments.